wp-content/themes/core/assets/js/config/i18n.js

```diff
@@ -0,0 +1 @@
+export const NLS = window.modern_tribe_i18n || {};
```

Comment:
The `wp.i18n` global object wasn't always available, so I opted to just port what we had in SQ1 for now. Without actual use cases, I don't have a vision for how this could be most helpful for Moose projects. Let me know if you have ideas @dpellenwood

Reply:
I'm pretty sure you can skip this global set up completely. If we need need to use strings in JS modules we can [import the in WP Core i18n package](https://developer.wordpress.org/block-editor/how-to-guides/internationalization/) within the specific JS module which will utilize it.